LevelComponents/EntitySet.cpp

```diff
@@ -193,6 +191,9 @@ namespace LevelComponents
     /// </summary>
     void EntitySet::ResetTile8x8Array()
     {
+        if (blankTile) // nulptr is convertible to bool
```

Comment:
How is blankTile set before calling this function? It seems like an odd way to trigger this logic.

Reply:
it is initialized as a nullptr in the header which will be done when creating the instance